user/troubleshooting/vm-troubleshooting.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: doc
+title: Suspend/Resume Troubleshooting
+permalink: /doc/vm-troubleshooting/
+redirect_from:
+- /doc/remove-vm-manually/
+---
+
+# VM troubleshooting #
+
+## VM Kernel troubleshooting ##
+
+In case of problems, you can access the VM console using `qvm-console-dispvm VMNAME` in dom0, then access the GRUB menu.
+You need to call it just after starting the VM (until `GRUB_TIMEOUT` expires); for example, in a separate dom0 terminal window.
+
+In any case you can later access the VM's logs (especially the VM console log `/var/log/xen/console/guest-VMNAME.log`).
+
+You can always set the kernel back to some dom0-provided value to fix a VM kernel installation.
+
+## Qubes starts, but no VMs load ##
+
+This issue may occur if a dom0 update is interrupted halfway through and/or a hard power off is done without shutting down Qubes, which results in files getting corrupted. 
+In this case, the best fix is to reinstall Qubes and restore your files from a backup. 
```

Comment:
This is nuclear option, in many cases there are less intrusive methods, depending on why exactly VMs do not start. Better option is to recommend trying to start some specific VM to see exact failure message and direct further steps based on that.